src/bluegreen.py

```diff
@@ -63,7 +64,13 @@ def post(self, url, body):
       "Authorization": "bearer " + self.token,
       "Content-Type": "application/x-www-form-urlencoded",
      }
-    conn = httplib.HTTPConnection(self.target)
+
+    conn = ""
+    if re.match('https', self.target):
```

Comment:
You could replace the regexp matching with a more robust solution, like `urlparse`. This suggestion supports Python 2 and 3:

```suggestion
    try:
        from urllib.parse import urlparse
    except:
        from urlparse import urlparse

    if urlparse(self.target).scheme == 'https':
```

You should also extract this piece of code to a separate function, to avoid replicating it.

Reply:
Good suggestion. =)

Reply:
Nice @ggarnier! @marcorosner could you please write a unit test for this use case?